Automattic/wc-calypso-bridge
# Hide Payments menu item for unsupported countries

### Changes proposed in this Pull Request:



This PR conditionally hides the Payments menu item for countries that do not support Woo Payments.

This is when Woo Payments adds the Payments menu item: https://github.com/Automattic/woocommerce-payments/blob/develop/includes/admin/class-wc-payments-admin.php#L205

Closes #1345 






### How to test the changes in this Pull Request:




1. Create a new Woo Express dev site (plan is not relevant).
2. Use a country that supports Woo Payments as the store's base (under Settings > WooCommerce > General) -- for example, US.
3. Ensure that the Payments menu item is visible. 
4. Use a country that doesn't support Woo Payments as the store's base (under Settings > WooCommerce > General) -- for example, South Africa.
5.  Ensure that the Payments menu item is visible. 
6. Apply the patch. 
7. The Payments menu item should be hidden. 
8. Use a country that supports Woo Payments as the store's base (under Settings > WooCommerce > General) -- for example, US.
9.  The Payments menu item should be visible. 



### Other information:




### FOR PR REVIEWER ONLY:
